### ISSUE TITLE
Using Execute() with PRAGMA causes exception

### ISSUE THREAD

**Issue by kiddailey** [sqlcipher/sqlcipher#470]
When executing PRAGMA statements, using the "Execute()" method results in a exception being thrown.  I discovered this while migrating from [sqlite-net](https://github.com/praeclarum/sqlite-net) v1.8.116 to the official Zetetic SQLCipher.  Note that sqlite-net does not suffer from this issue.

I have not tested to see if this issue extends beyond iOS NET and the zetetic sqlite net base.

I did a cursory search on the discussion forums and here on GitHub and did not see mention of this issue.

### Expected Behavior

PRAGMA operation completes normally.

### Actual Behavior

An exception is thrown.  No additional information is provided. Attempting to catch the exception results in an exception that has a null value.  In the case of iOS, logging provides no additional information.  In my case, the error was occurring on startup and iOS was logging an exception the "apps require a main window" exception which made this a little tricky to figure out.

### Steps to Reproduce

In my tests, any PRAGMA call causes this issue.  As an example, the following causes the exception:

```
var dataContext = new SQLiteConnection(databasePath);
dataContext.Execute(string.Format("PRAGMA cipher_license = '{0}';", "<license code>"));
```

A workaround is to use the ExecuteScaler<int>() method, which does works as expected:

```
var dataContext = new SQLiteConnection(databasePath);
dataContext.ExecuteScalar<int>(string.Format("PRAGMA cipher_license = '{0}';", "<license code>"));
```

SQLCipher version:

sqlcipher-ios-net-4.5.4

**Comment by developernotes** [comment]
Hi @kiddailey,

This is expected behavior because `PRAGMA cipher_license = '...';` returns a value. It sounds as if you may have a commercial license to SQLCipher. If so, you will see in the bundled demo application included with your download package the utilization of `ExecuteScalar<int>(....)` when applying the license code. This applies to all `PRAGMA` statements which return a value. For example, if you wish to enable WAL mode via pragma, you would need to execute the follow, because the return value is a string.

```
var mode = _connection.ExecuteScalar<string>("PRAGMA journal_mode = wal;");
```

Would you take a look at the bundled demo application and let us know if you have any additional questions?

**Comment by kiddailey** [comment]
Thanks for the reply!  Yes, I did look at the bundled application which is what helped me figure out what was going on :)  

For what it's worth, I just used the license setting as an example.  It also triggers exceptions with other PRAGMA calls (e.g. setting the key, setting the pbkdf2 iterations, etc.).

I didn't really have any questions.  It just seemed like unexpected behavior because the exception didn't have any details about what caused it, and I at least wanted to document for anyone else -- particularly when sqlite-net doesn't have the same issue.

Perhaps it would be nice if the thrown exception at least had a message to indicate the issue, but not a biggie.  Thanks again.
